wiki/release.md

```diff
@@ -0,0 +1,30 @@
+# Releases
+
+## Release workflow
+
+the flow should be goes as following
+
+1. Each release is developed against branch at the beginning and this is branched from development `i.e development-2.0`
+2. All `pull requests` should be to this branch no commits goes to `development` directly
+3. Everything on development should reflect the latest release
+4. Once release branch is merged to `master`, a tag should be created and pushed to the repo, tag should be in format `v*` format for the release workflow to work
+5. Our github workflow is fired when a tag is pushed
+6. The workflow basicly runs goreleaser to create a new release which creates the different release assed required to run the provider of different platforms `i.e windows,linux and mac`
+7. Once the release is done the terraform registry always watch the repo for new releases and when it finds a new release it publishes a new provider version
+
+## Notes
+
+As you may know the grid terraform provider depends on zos and substrate client so on new releases on those repos, this should be reflected here
+
+### Updating zos version
+
+1. This basicly done as any go dependency update the required version in go.mod and if there are changes required on the code should be fixed for example `go get https://github.com/threefoldtech/zos@d9c7fe2`
+
+### Updating substrate client
+
+1. In the current design we have substrate pkg for each environment as submodules(this should be changed soon, we should use provider versions instead)
+2. When we want to update substrate-client for any environment for exampe for dev env we go to pkgs/substrates/substrate-dev and checkout the commit we want to update to and then we do the required changes on code
```

Comment:
exampe -> example, env -> environment, and let's add backticks around pkgs/substrates/substrate-dev

Reply:
also please add the commands used